### PR TITLE
Update git versioning

### PR DESCRIPTION
**Pull request by networkfusion**
## Motivation and Context
Nerdbank GitVersioning was behind.

## Types of changes

- [ ] Improvement (non-breaking change that improves a feature, code or algorithm)
- [ ] Bug fix (non-breaking change which fixes an issue with code or algorithm)
- [ ] New feature (non-breaking change which adds functionality to code)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Config and build (change in the configuration and build system, has no impact on code or features)
- [x] Dependencies (update dependencies and changes associated, has no impact on code or features)
- [ ] Unit Tests (add new Unit Test(s) or improved existing one(s), has no impact on code or features)
- [ ] Documentation (changes or updates in the documentation, has no impact on code or features)

## Checklist:

- [ ] My code follows the code style of this project (only if there are changes in source code).
- [ ] My changes require an update to the documentation (there are changes that require the docs website to be updated).
- [ ] I have updated the documentation accordingly (the changes require an update on the docs in this repo).
- [ ] I have read the [CONTRIBUTING](https://github.com/nanoframework/.github/blob/master/CONTRIBUTING.md) document.
- [ ] I have tested everything locally and all new and existing tests passed (only if there are changes in source code).
- [ ] I have added new tests to cover my changes.

### CODE REVIEW

File: System.Device.Dac/System.Device.Dac.nfproj [hunk 1]
```diff
@@ -64,11 +64,12 @@
       <ProjectConfigurationsDeclaredAsItems />
     </ProjectCapabilities>
   </ProjectExtensions>
-  <Import Project="..\packages\Nerdbank.GitVersioning.3.1.91\build\Nerdbank.GitVersioning.targets" Condition="Exists('..\packages\Nerdbank.GitVersioning.3.1.91\build\Nerdbank.GitVersioning.targets')" />
+  <Import Project="..\packages\Nerdbank.GitVersioning.3.5.113\build\Nerdbank.GitVersioning.targets" Condition="Exists('..\packages\Nerdbank.GitVersioning.3.5.113\build\Nerdbank.GitVersioning.targets')" />
```
Comment (josesimoes) [inline System.Device.Dac/System.Device.Dac.nfproj:67]: Sorry Robin: this one is missing the props...

Comment (networkfusion) [reply System.Device.Dac/System.Device.Dac.nfproj:67]: Are you sure? I have just checked against `Spi` and `Collections` and the O/P at this line is the same...

Comment (networkfusion) [reply System.Device.Dac/System.Device.Dac.nfproj:70]: Ok... I have found what you mean.
will add in a min,

Comment (josesimoes) [reply System.Device.Dac/System.Device.Dac.nfproj:71]: Pretty sure! I've just fixed it.

**Comment by networkfusion** [comment]
Beat me too it again 😛

**Comment by networkfusion** [comment]
Obsolete.

**Pull request closed without merging**